src/api.js

```diff
@@ -174,6 +174,13 @@ export const MOCK_EVENT = {
 	yes_rsvp_count: 23
 };
 
+export const MOCK_BEST_TOPIC = {
```

Comment:
That's fine, though I wouldn't characterize there being nothing 'best-specific` about it -- seeing that this is the language documented for the corresponding API method (https://www.meetup.com/meetup_api/docs/find/topic_categories/).  

> Authenticated members may request "best_topics" using the "fields" request parameter.

Both the input field and output for API is called 'best_topics':